src/Search.elm

```diff
@@ -511,7 +516,7 @@ viewPager outMsg model result path =
             ]
         , li
             [ classList
-                [ ( "disabled", model.from + model.size >= result.hits.total.value )
+                [ ( "disabled", result.hits.total.value == 10000 || model.from + model.size >= result.hits.total.value )
```

Comment:
It's not really disabled, it can still be used by clicking on it, and it goes to the proper last page.